src/actions/background/Background.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Defines the background color to use instead of transparent background areas or when resizing with padding.
+ *
+ * **Learn more**:
+ * <a href="https://cloudinary.com/documentation/image_transformations#setting_background_color" target="_blank">
+ * Setting background for images</a> |
+ * <a href="https://cloudinary.com/documentation/video_manipulation_and_delivery#background_color" target="_blank">
+ * Setting background for videos</a>
+ *
+ * @api
```

Comment:
Unfortunately, we can't just copy the docstrings from PHP, certain things need to change.

Also, please make sure this looks proper when you build the docs (npm run build:docs).
Pay attention to line breaks and spaces.

Also, `Background` should be part of the Actions namespace, when building the docs ensure there's an `Actions.Background` entry (under namespaces)